roles/telegraf/files/run-telegraf.sh

```diff
@@ -369,7 +369,7 @@ function main {
       write_prometheus "${consul_prefix}" "${type}" "${consul_template_config}" "${consul_config}" "${config_override_dir}"
     fi
 
-    supervisorctl signal SIGHUP consul
+    systemctl kill -s SIGHUP consul
     supervisorctl signal SIGHUP consul-template
```

Comment:
Update this line too.

Reply:
Do you mean, change the run-consul-template script to start consul-template with systemd instead of supervisord, too?

It's not a requirement for this upgrade, but I suppose we could consolidate our services.

Reply:
Yep. Might as well get rid of supervisord while we are at it.